mailpoet/webpack.config.js

```diff
@@ -562,10 +554,9 @@ const marketingOptinBlock = Object.assign({}, wpScriptConfig, {
     extensions: ['.js', '.jsx', '.ts', '.tsx'],
   },
   plugins: [
-    ...wpScriptConfig.plugins.filter(
-      (plugin) =>
-        plugin.constructor.name !== 'DependencyExtractionWebpackPlugin',
-    ),
+    ...wpScriptConfig.plugins.filter((plugin) => {
+      return plugin.constructor.pluginName === 'mini-css-extract-plugin';
+    }),
```

Comment:
Could you please elaborate a bit on why this change was needed? Maybe even a comment in the code explaining why we filter the plugins like that could be useful.

Reply:
`CopyWebpackPlugin` was the one causing the problems and with the previous config we were allowing that to be used from `wpScriptConfig` even though we were adding our custom one as well.